spec/src/main/asciidoc/configsources.asciidoc

```diff
@@ -35,8 +35,6 @@ It can also be used to implement a drop-in configuration approach.
 Simply create a jar containing a `ConfigSource` with a higher ordinal and override configuration values in it.
 If the jar is present on the classpath then it will override configuration values from <<ConfigSource,ConfigSources>> with lower `ordinal` values.
 
-The ordinal for property file based <<ConfigSource,ConfigSources>> can be configured using the key `config_ordinal` inside the property file.
```

Comment:
With the solution I propose, there's still a way to opt out of using the `config_ordinal`, because the source has complete control over what it returns from the `getValue()` method. If it's really necessary to avoid reading the `config_ordinal` property, the `getValue()` method can treat this property in a special way and e.g. return the value of `getOrdinal()` intad of reading it from configuration.